packages/@expo/cli/src/login/index.ts

```diff
@@ -27,6 +29,7 @@ export const expoLogin: Command = async (argv) => {
         `-u, --username <string>  Username`,
         `-p, --password <string>  Password`,
         `--otp <string>           One-time password from your 2FA device`,
+        `-s, --sso                Log in with SSO`,
```

Comment:
I assume having the argument listed here is required for the parser to work? We hid this in the eas-cli PR. Is it not possible to hide here as well?